### PR TITLE
[Documentation] Update overview section

### PR DESCRIPTION
**Pull request by Batta32**
Related to 3689

### Purpose
*What is the context of this pull request? Why is it being done?*
The [overview](https://github.com/microsoft/botframework-solutions/tree/master/docs/_docs/overview) section of the public documentation is outdated

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
- Remove unnecesary Release Notes for release beta 0.7/0.8
- Remove redundant information
- Fix broken/empty links

The updated documents are the followings:
- docs/_docs/overview/skills.md
- docs/_docs/overview/virtual-assistant-solution.md
- docs/_docs/overview/whats-new/0.7-beta/Summary.md
- docs/_docs/overview/whats-new/0.7-beta/release-notes.md
- docs/_docs/overview/whats-new/0.8-beta/release-notes.md 

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
It was rendered manually in order to check the result of the changes

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [X] I have updated related documentation

### CODE REVIEW

File: docs/_docs/overview/whats-new/0.7-beta/Summary.md [hunk 1]
```diff
@@ -24,12 +24,12 @@ Virtual Assistant Core has many new features that have been added to follow the
 ### Bot Framework 4.6 Support
 {:.no_toc}
 
-Virtual Assistant is updated to Bot Framework 4.6.  Some new capabilities of the SDK are highlighted in this document, but more details of Bot Framework 4.6 can be found [here](https://github.com/microsoft/botframework#Bot-Framework-SDK-v4).
+Virtual Assistant is updated to Bot Framework 4.6.  Some new capabilities of the SDK are highlighted in this document, but more details of Bot Framework 4.6 can be found [here](https://github.com/microsoft/botframework-sdk).
```
Comment (DiegoCardozo94) [inline docs/_docs/overview/whats-new/0.7-beta/Summary.md:27]: Doble space in this line?

File: docs/_docs/overview/whats-new/0.7-beta/Summary.md [hunk 2]
```diff
@@ -24,12 +24,12 @@ Virtual Assistant Core has many new features that have been added to follow the
...
 
 ### Language Generation (LG) Support
 {:.no_toc}
 
-Microsoft has added support of the new [Language Generation (LG)](https://github.com/Microsoft/BotBuilder-Samples/tree/master/experimental/language-generation) features that have been added to the [Bot Framework 4.6](https://github.com/microsoft/botframework#Bot-Framework-SDK-v4) allowing for richer dialogs that have more dynamic, natural responses.  Microsoft has also incorporated LG into the Virtual Assistant Skills.
+Microsoft has added support of the new [Language Generation (LG)](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-concept-language-generation?view=azure-bot-service-4.0&tabs=csharp) features that have been added to the [Bot Framework 4.6](https://github.com/microsoft/botframework-sdk) allowing for richer dialogs that have more dynamic, natural responses.  Microsoft has also incorporated LG into the Virtual Assistant and the Skills.
```
Comment (DiegoCardozo94) [inline docs/_docs/overview/whats-new/0.7-beta/Summary.md:32]: Doble space in this line?

**Review by fran893 — APPROVED** [review]

**Review by DiegoCardozo94 — COMMENTED** [review]
Can you check these lines? The rest LGTM!

File: docs/_docs/overview/whats-new/0.7-beta/Summary.md (hunk 2 above)
Comment (Batta32) [reply docs/_docs/overview/whats-new/0.7-beta/Summary.md:32]: Agree with you. Thanks!

File: docs/_docs/overview/whats-new/0.7-beta/Summary.md (hunk 1 above)
Comment (Batta32) [reply docs/_docs/overview/whats-new/0.7-beta/Summary.md:27]: Agree with you. Thanks!

**Review by FedericoBernal — APPROVED** [review]
LGTM

**Review by matiasroldan6 — APPROVED** [review]

File: docs/_docs/overview/skills.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ The following Skill samples are available out of the box, each with deployment s
 ### Experimental Skills
 
 These experimental Bot Framework Skills are early prototypes to help bring Skill concepts to life for demonstrations and proof-of-concepts.
-By their very nature these Skill are not complete, with only English support. If you have any feedback on these Skills, please [open a new issue](https://github.com/microsoft/botframework-solutions/issues/new/choose) on the Bot Framework Solutions repository.
+By their very nature these Skill are not complete, with only English support. If you have any feedback on these Skills, please [open a new issue](https://github.com/microsoft/botframework-skills/issues/new/choose) on the Bot Framework Skills repository.
```
Comment (matiasroldan6) [inline docs/_docs/overview/skills.md:36]: *these Skills

Comment (Batta32) [reply docs/_docs/overview/skills.md]: Thanks!

**Pull request merged**